DecipherNow/gm-fabric-dashboard
# feat: allow users to access service view for service with no metrics

Resolves #1344 

## Proposed Changes

  - Passed the serviceIsMetered prop down from the top level router so that GMServiceTableLineItem.js can decide whether to disable the link
  - Added an icon and a tooltip to GMServiceTableLineItem.js
  - Customized Tooltip to allow for disabling and custom styling
  
## How to Test

  - From both cards and list view, navigate to Grace Hopper. You should see the unmetered icon next to the instance id as well as a tooltip on hover.
- Navigate to a metered service and confirm there is no tooltip or icon
- Navigate to: http://localhost:3000/#/Grace%C2%B7Hopper%C2%B7Battleship%C2%B7Control%C2%B7Program/1.0/ee0fa3669fea7e9a0adea649c46bca56 
       You should be redirected back to the service view and see a notification
